dts/bindings/arm/nordic,nrf21540-gpio.yaml

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) 2020 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+
+description: >
+    This is a representation of the nRF21540 Radio Front-End module
+compatible: "nordic,nrf21540_gpio"
```

Comment:
This should use `-` instead of `_`.
And since this binding contains not only strictly GPIO related properties but also time and gain values, maybe instead of `gpio` it should use something else in the compatible string, e.g. `fem`?

Reply:
nRF21540 can be controlled by two interfaces: GPIO or SPI. This binding describes controlling by GPIO, that's the reason of `_gpio` suffix. When SPI is implemented, another binding `nordic/nrf21540_spi` should be created.

Reply:
What about antenna select pin in that case ? Where will they be configured ?